shell/platform/darwin/ios/framework/Source/FlutterViewControllerTest.mm

```diff
@@ -206,23 +209,34 @@ - (void)testStartKeyboardAnimationWillInvokeSetupKeyboardAnimationVsyncClient {
   FlutterViewController* viewControllerMock = OCMPartialMock(viewController);
   viewControllerMock.targetViewInsetBottom = 100;
   [viewControllerMock startKeyBoardAnimation:0.25];
-  OCMVerify([viewControllerMock setupKeyboardAnimationVsyncClient]);
+
+  CAAnimation* keyboardAnimation =
+      [[viewControllerMock keyboardAnimationView].layer animationForKey:@"position"];
+
+  OCMVerify([viewControllerMock setupKeyboardSpringAnimationIfNeeded:keyboardAnimation]);
```

Comment:
The `testStartKeyboardAnimationWillInvokeSetupKeyboardSpringAnimationIfNeeded ` is moved to `testFlutterViewControllerStartKeyboardAnimationWillCreateVsyncClientCorrectly`